src/openrct2/rct2/S6Exporter.cpp

```diff
@@ -255,6 +255,24 @@ static void scenario_fix_ghosts(rct_s6_data* s6)
     }
 }
 
+template<typename T> static void ExportObjectList(IObjectManager& objMgr, T& objects, ObjectType type, size_t maxEntries)
+{
+    for (size_t i = 0; i < maxEntries; i++)
+    {
+        auto& dst = objects[i];
+
+        auto* object = objMgr.GetLoadedObject(type, i);
+        if (object == nullptr || object->GetObjectEntry() == nullptr)
+        {
+            std::memset(&dst, 0xFF, sizeof(dst));
```

Comment:
add a comment that this is for save compatibility that it has an entry of 0xFF, ...